Makefile

```diff
@@ -98,6 +98,7 @@ mock:
 ifneq ($(HAS_SERVER),)
 	go install github.com/golang/mock/mockgen
 	mockgen -destination server/mscalendar/mock_mscalendar/mock_mscalendar.go github.com/mattermost/mattermost-plugin-mscalendar/server/mscalendar MSCalendar
+	mockgen -destination server/mscalendar/mock_mscalendar/mock_welcomer.go github.com/mattermost/mattermost-plugin-mscalendar/server/mscalendar Welcomer
```

Comment:
In this case, I mocked the Welcomer because the test was calling `AfterSuccessfullyConnect`, which makes several operations that made the test less unitary and more confusing. Therefore, instead of using the "proper welcomer" already mocked on the MSCalendar, I preferred to mock it, and at some point maybe test it in a more unitary way.